ioctl/newcmd/account/account_test.go

```diff
@@ -0,0 +1,314 @@
+	"github.com/iotexproject/iotex-core/testutil"
+)
+
+const (
+	testPath = "ksTest"
+)
+
+func TestNewAccountCmd(t *testing.T) {
+	require := require.New(t)
+	ctrl := gomock.NewController(t)
+	client := mock_ioctlclient.NewMockClient(ctrl)
+	client.EXPECT().SelectTranslation(gomock.Any()).Return("mockTranslationString", config.English).AnyTimes()
+	client.EXPECT().Config().Return(config.ReadConfig).AnyTimes()
+	cmd := NewAccountCmd(client)
+	result, err := util.ExecuteCmd(cmd)
+	require.NotNil(result)
+	require.NoError(err)
+}
+
+func TestSign(t *testing.T) {
+	require := require.New(t)
+	testWallet := filepath.Join(os.TempDir(), testPath)
+	require.NoError(os.MkdirAll(testWallet, os.ModePerm))
+	defer testutil.CleanupPath(t, testWallet)
+	config.ReadConfig.Wallet = testWallet
+
+	ks := keystore.NewKeyStore(config.ReadConfig.Wallet, keystore.StandardScryptN, keystore.StandardScryptP)
+	require.NotNil(ks)
+
+	// create accounts
+	nonce := strconv.FormatInt(rand.Int63(), 10)
+	passwd := "3dj,<>@@SF{}rj0ZF#" + nonce
+
+	account, err := ks.NewAccount(passwd)
+	require.NoError(err)
+	addr, err := address.FromBytes(account.Address.Bytes())
+	require.NoError(err)
+	require.True(IsSignerExist(addr.String()))
+
+	result, err := Sign(addr.String(), passwd, "abcd")
+	require.NoError(err)
+	require.NotEmpty(result)
+
+	result, err = Sign(addr.String(), passwd, "0xe3a1")
+	require.NoError(err)
+	require.NotEmpty(result)
+
+	result, err = Sign(addr.String(), passwd, "abc")
+	require.Error(err)
+	require.Contains(err.Error(), "odd length hex string")
+	require.Equal("", result)
+
+	result, err = Sign("hdw::aaaa", passwd, "0xe3a1")
+	require.Error(err)
+	require.Contains(err.Error(), "invalid HDWallet key format")
+	require.Equal("", result)
+
+	result, err = Sign(addr.String(), "123456", "abcd")
+	require.Error(err)
+	require.Contains(err.Error(), "could not decrypt key with given password")
+	require.Equal("", result)
+
+	result, err = Sign("bace9b2435db45b119e1570b4ea9c57993b2311e0c408d743d87cd22838ae892", "123456", "test")
+	require.Error(err)
+	require.Contains(err.Error(), "invalid address")
+	require.Equal("", result)
+
+	prvKey, err := PrivateKeyFromSigner(addr.String(), passwd)
+	require.NoError(err)
+	require.NotNil(prvKey)
+
+	prvKey, err = PrivateKeyFromSigner(addr.String(), "123456")
+	require.Error(err)
+	require.Contains(err.Error(), "could not decrypt key with given password")
+	require.Nil(prvKey)
+}
+
+func TestAccount(t *testing.T) {
+	require := require.New(t)
+	testWallet := filepath.Join(os.TempDir(), testPath)
+	require.NoError(os.MkdirAll(testWallet, os.ModePerm))
+	defer testutil.CleanupPath(t, testWallet)
+	config.ReadConfig.Wallet = testWallet
+
+	ks := keystore.NewKeyStore(config.ReadConfig.Wallet, keystore.StandardScryptN, keystore.StandardScryptP)
+	require.NotNil(ks)
+
+	// create accounts
+	nonce := strconv.FormatInt(rand.Int63(), 10)
+	passwd := "3dj,<>@@SF{}rj0ZF#" + nonce
```

Comment:
move into a test util function?

Reply:
yes, these codes generate a true account , and is used by other test functions.